### PR TITLE
Radio-group fieldset panel indent

### PR DESCRIPTION
**Pull request by JoeChapman**
- add optional panel-indent classname to partial
- get panel-indent value from fields[key]

### CODE REVIEW

**Comment by lennym** [comment]
This seems way too specific to me. Having options for specific classes for specific use cases feels nasty. I'd be much happier if it implemented `className` in the same way as the text inputs.

See https://github.com/UKHomeOffice/passports-template-mixins/blob/master/lib/template-mixins.js#L85

**Comment by JoeChapman** [comment]
How about a `fieldsetClassName` property, which is assigned to the `fieldset`?

**Comment by lennym** [comment]
To be honest, I'd be happier with just `className` as a property on the radio group, since `fieldset` is the containing tag for a radio group right now, but it doesn't *need* to be, so probably shouldn't create a coupling between the tag name and the config property.

**Comment by lennym** [comment]
This looks much nicer :-)

File: lib/template-mixins.js [hunk 1]
```diff
@@ -95,12 +95,13 @@ module.exports = function (t, fields, options) {
...
-    function optionGroup(key) {
-        return {
+    function optionGroup(key, extension) {
+        extension = extension || {};
+        return _.extend(extension, {
             'key': key,
             'error': this.errors && this.errors[key],
-            'legend': fields[key].legend && fields[key].legend.value,
-            'options': _.map(fields[key].options, function (obj) {
+            'legend': fields[key] && fields[key].legend && fields[key].legend.value,
```
Comment (lennym) [inline lib/template-mixins.js]: What was the behaviour here before? I'm not 100% sure that we don't want the template to error out if you attempt to reference an undefined field.

Comment (JoeChapman) [inline lib/template-mixins.js]: simple test such as the following will fail:
https://github.com/UKHomeOffice/passports-template-mixins/blob/master/test/spec.index.js#L55

So I mirrored this defensive approach: https://github.com/UKHomeOffice/passports-template-mixins/blob/master/lib/template-mixins.js#L82

**Pull request merged**